testing/upload_validation.ts

```diff
@@ -390,6 +391,25 @@ const defaultAuthenticationValidators: Record<AuthenticationType, z.ZodTypeAny>
     tokenPrefix: z.string().optional(),
     ...baseAuthenticationValidators,
   }),
+  [AuthenticationType.MultiHeaderToken]: zodCompleteStrictObject<MultiHeaderTokenAuthentication>({
+    type: zodDiscriminant(AuthenticationType.MultiHeaderToken),
+    headers: z
+      .array(
+        zodCompleteStrictObject<MultiHeaderTokenAuthentication['headers'][number]>({
+          name: z.string(),
+          description: z.string(),
+          tokenPrefix: z.string().optional(),
+        }),
+      )
+      .refine(
+        headers => {
+          const keys = headers.map(header => header.name);
```

Comment:
Might be worth making `head.name` lowercase here since HTTP headers ought to be case insensitive and it would be weird to have both `Header-Foo` and `header-foo` in the list

Reply:
Yep great point, done.